server/src/user.h

```diff
@@ -0,0 +1,27 @@
+#ifndef USER_H
+#define USER_H
+
+#include <QString>
+
+struct User
+{
+	User()
```

Comment:
зачем там много строчек
```suggestion
	User() {}
```